### ISSUE TITLE
Newly cloned project won't compile

### ISSUE THREAD

**Issue by AndyCErnst** [prismicio/scala-kit#48]
I just tried pulling this down and I'm getting when running `sbt compile`. I'm guessing this dependency broke?
```
[error] /scala-kit/project/Build.scala:57: object Plugin is not a member of package net.virtualvoid.sbt.graph
[error]     settings = BuildSettings.buildSettings ++ net.virtualvoid.sbt.graph.Plugin.graphSettings ++ Seq(
```

**Comment by srenault** [comment]
Yep, 
Our `sbt-dependency-graph` is outdated and either need to be removed or updated.
I'll do this ASAP.

Thanks for your report!

**Comment by srenault** [comment]
Ok it's fixed here 27986ce952e0f64d4d542a8a560d40ef1dcaa2c3

**Comment by AndyCErnst** [comment]
Fantastic, thanks.
